src/sugar3/graphics/radiopalette.py

```diff
@@ -22,6 +22,10 @@
 
```

Comment:
Please add a short description about the top of the file and embed your example code.